docs/tutorial/part-two/index.md

```diff
@@ -296,36 +292,36 @@ export default () => (
 )
 ```
 
-> Tip: Generally, if you use a component in multiple places on a site, it should be in its own module file in the `components` directory. But, if it's used only in one file, create it inline.
+> नोट: आम तौर पर, यदि आप किसी साइट पर कई स्थानों पर एक कौम्पोनॅन्ट का उपयोग करते हैं, तो यह `Components` फोल्डर में अपनी मॉड्यूल फ़ाइल में होना चाहिए। लेकिन, अगर यह केवल एक फ़ाइल में उपयोग किया जाता है, तो इसे इनलाइन बनाएं।
 
-The finished page should now look like:
+समाप्त पेज अब ऐसा दिखना चाहिए:
 
-![User list page with CSS modules](css-modules-userlist.png)
+![सीएसएस मॉड्यूल के साथ उपयोगकर्ता सूची पेज](css-modules-userlist.png)
 
 ### CSS-in-JS
 
-CSS-in-JS is a component-oriented styling approach. Most generally, it is a pattern where [CSS is composed inline using JavaScript](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js).
+CSS-in-JS एक कौम्पोनॅन्ट-ओरियेनटेड स्टाइलिंग दृष्टिकोण है। ज्यादातर आम तौर पर, यह एक पैटर्न है जहां [सीएसएस जावास्क्रिप्ट का उपयोग करके इनलाइन बनाया जाता है](https://reactjs.org/docs/faq-styling.html#what-is-css-in-js)
```

Comment:
`(https://reactjs.org/docs/faq-styling.html#what-is-css-in-js)` => `(https://reactjs.org/docs/faq-styling.html#what-is-css-in-js)।`

missing `purn viram`